tests/topotests/lib/bgp.py

```diff
@@ -5638,3 +5638,22 @@ def configure_bgp_soft_configuration(tgen, dut, neighbor_dict, direction):
             )
         )
         return True
+
+
+def bgp_configure_prefixes(router, asn, safi, prefixes, vrf=None, update=True):
+    """
+    Configure the bgp prefixes.
+    """
+    withdraw = "no " if not update else ""
+    vrf = " vrf {}".format(vrf) if vrf else ""
+    for p in prefixes:
+        ip = ipaddress.ip_network(p)
+        cmd = [
+            "conf t\n",
+            "router bgp {}{}\n".format(asn, vrf),
```

Comment:
Can we use string interpolation with `f""` (https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498)? Easier to read and maintain.

Reply:
done